docs/docs/dev_docs/getting_started/cli.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Aztec CLI
+---
+
+## Introduction
+
+The Aztec CLI is a tool designed to enable a user to interact with the Aztec Network. Here we will provide a walk-through demonstrating how to use the CLI to deploy and test contracts on the [Aztec Sandbox](../sandbox/main.md).
+
+## Requirements
+
+The Aztec CLI is an npm package so you will need to have installed node.js >= 18. This tutorial will use the Aztec Sandbox so you should first set up the sandbox using the link above.
+
+To install the Aztec CLI simply:
+
+`yarn global add @aztec/cli`
```

Comment:
Also requires `yarn` as a dependency.
If we were feeling fancy, we could provide commands for both yarn and npm, via docusaurus tabs (https://docusaurus.io/docs/markdown-features/code-blocks#multi-language-support-code-blocks). But not essential for this PR :)